macs2.wdl

```diff
@@ -30,8 +30,15 @@ task PeakCalling {
         String sampleName
         String format = "AUTO"
         Boolean nomodel = false
+        String gensz = "hs"
+        Int extsize
+        Int shiftsize = -1*round(extsize/2)
+        Float pval_thres = 0.01
+        Boolean bdg = true
+        String keepdup = "auto"
+        Boolean callsummits = true
         Int timeMinutes = 600  # Default to 10 hours
-        String memory = "8GiB"
```

Comment:
Please keep it at GiB. 8 * 1024 * 1024 * 1024 is significantly more than 8 * 1000 * 1000 * 1000. (Cromwell does not make the distinction and always chooses the larger option, but miniwdl does)